{"name": "Проверка логина с неверными данными", "status": "passed", "description": "\n                                1. Генерим случайные данные;\n                                3. Оправляем запрос на логин;\n                                3. Проверяем ответ;\n                                ", "start": 1718224961084, "stop": 1718224961411, "uuid": "e97be610-42f2-4d8d-814f-4454fbc58539", "historyId": "607399e3c691e986195ddf64bb28a636", "testCaseId": "607399e3c691e986195ddf64bb28a636", "fullName": "tests.test_login_user.TestCreateUser#test_login_incorrect_data", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "otemorygame"}, {"name": "thread", "value": "73580-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}